packages/assets/lib/external/polkadot_directory_repo.js

```diff
@@ -0,0 +1,53 @@
+/* @license Copyright 2023 @polkadot-cloud/library authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+import fs from "fs";
```

Comment:
To keep the codebase consistent shall we just use `fs/promises`? I found that syntax looks a lot cleaner too.

```suggestion
import fs from "fs/promises";
```